dataland-frontend/src/components/general/DatasetsTabMenu.vue

```diff
@@ -60,19 +60,21 @@ export default defineComponent({
     };
   },
   created() {
-    void checkIfUserHasRole(KEYCLOAK_ROLE_REVIEWER, this.getKeycloakPromise).then((hasUserReviewerRights) => {
-      this.tabs[2].isVisible = hasUserReviewerRights;
-    });
+    checkIfUserHasRole(KEYCLOAK_ROLE_REVIEWER, this.getKeycloakPromise)
+      .then((hasUserReviewerRights) => {
+        this.tabs[2].isVisible = hasUserReviewerRights;
+      })
+      .catch((error) => console.log(error));
   },
   methods: {
     /**
      * Routes to companies page when AVAILABLE DATASET tab is clicked
      * @param event the event containing the index of the newly selected tab
      * @param event.index the index of the tab element
      */
-    handleTabChange(event: { index: number }): void {
+    async handleTabChange(event: { index: number }): void {
```

Comment:
We did not change the three issues ("This assertion is unnecessary since it does not change the type of the expression.") for LksgPanel.vue and DatasetDisplayStatusIndicator.vue for we feel that those are false positives on sonar side. Further eslint throws an error when those expressions are removed.